app/components/tab_navigation/style.scss

```diff
@@ -85,6 +85,10 @@
   position: relative;
   text-decoration: none;
 
+  &:focus:before {
+    height: 7px; // Bar needs to change thickness so that we don’t rely on colour alone.
```

Comment:
Consider setting height at 8px here so it's more noticeable.

Reply:
Think I'll leave as is for now - you can see it change size, and the *current* tab is probably the least important. Manage has also now put in a PR using these styles.